#SAK 13922 - FCK editor fix
=svnm= https://source.sakaiproject.org/svn/reference/trunk@49120 reference

#Fill in the SAK or something unique, leave it like this if theres no real patch
--- /dev/null
+++ SAK-13922.tmp
@@ -0,0 +1 @@
+ 
